index.html

```diff
@@ -2090,8 +2124,8 @@ <h3 id="applying">
           <code>orientation</code> member
         </h3>
         <p>
-          The <dfn id="member-orientation"><code>orientation</code>
-          member</dfn> is a <a>string</a> that serves as the <a>default
+          The <dfn><code>orientation</code></dfn>
```

Comment:
s/`<dfn><code>orientation</code></dfn>`/`<dfn>orientation</dfn> member`

As above, the `dfn` for this should occur inside a `h3` element in its own section.

Reply:
The above applies for all the things below.

Reply:
yes I can remove <code> from inside <dfn> and <a>, doing